dask_kubernetes/operator/customresources/daskautoscaler.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: daskautoscalers.kubernetes.dask.org
+spec:
+  scope: Namespaced
+  group: kubernetes.dask.org
+  names:
+    kind: Autoscaler
```

Should this be the title case version of the `singular` like our other CRDs are?

```suggestion
    kind: DaskAutoscaler
```